keras_cv/layers/preprocessing/color_jitter_test.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.color_jitter import ColorJitter
+
+
+class ColorJitterTest(tf.test.TestCase):
```

Comment:
Can we add tests to:

with_labels_test

serialization_test?  Thanks!

Reply:
@LukeWood can u elaborate a bit? or a reference link would be nice.

Reply:
Okay, I updated with_labels_test and serializaiton_test. The serialization test passes successfully but with_labels_test gives a 2/3 failed error, mainly on the randcom_cut layer.